src/test/java/com/baidu/hugegraph/api/BatchUpdateElementApiTest.java

```diff
@@ -173,6 +173,28 @@ public void testVertexBatchUpdateStrategyEliminate() {
         assertBatchResponse(vertices, "list", "old");
     }
 
+    @Test
+    public void testVertexBatchUpdateStrategyOverride() {
+        BatchVertexRequest req = batchVertexRequest("price", -1, 1,
+                                                    UpdateStrategy.OVERRIDE);
+
+        List<Vertex> vertices = vertexAPI.update(req);
+        assertBatchResponse(vertices, "price", 1);
+
+        req = batchVertexRequest("list", "old", "new", UpdateStrategy.OVERRIDE);
```

Comment:
there is always only one element in the list, is it expected and reasonable?

Reply:
`OVERRIDE`  is used for this case:
> Person have 3 props: `name, price, list` ,and **only** `price` have update strategy
> 1. Old value from backend : `tom, -1, [oldStr1, oldStr2](list)` 
> 2. New value1 from json : `tom, 1, [newStr1, newStr2]`
> 3. New value2 from json : `tom, null, null`
> 4. After update , get: `tom, 1, [newStr1, newStr2]`

however, if we don't set update strategy for list, we will get `tom, 1`  and lose the `list` property.

Reply:
That's OK,  if we can override the list as a whole.